test/e2e/velero_utils.go

```diff
@@ -327,7 +356,11 @@ func veleroCreateBackupLocation(ctx context.Context,
 // veleroAddPluginsForProvider determines which plugins need to be installed for a provider and
 // installs them in the current Velero installation, skipping over those that are already installed.
 func veleroAddPluginsForProvider(ctx context.Context, veleroCLI string, veleroNamespace string, provider string) error {
-	for _, plugin := range getProviderPlugins(provider) {
+	version, err := getVeleroVersion(ctx, veleroCLI, true)
+	if err != nil {
+		return errors.WithMessage(err, "Failed to get velero version")
+	}
+	for _, plugin := range getProviderPlugins(version, provider) {
```

Comment:
Use the user provided `plugins` first, if it isn't provided, then call `getProviderPlugins`